pyaedt/desktop.py

```diff
@@ -363,7 +368,12 @@ def __init__(
                 if StrictVersion(version_key) < StrictVersion("2022.2.0") or not settings.use_grpc_api:
                     print("Launching PyAEDT outside Electronics Desktop with CPython and Pythonnet")
```

Comment:
```suggestion
                    print("Launching PyAEDT outside Electronics Desktop with CPython and Python.Net.")
```

Reply:
Hi @PipKat , while I don't mind committing the small change to this print statement, it is outside the scope of changes I contributed to the code.  I'd appreciate the (somewhat redundant) affirmation that it is ok to proceed with this commit.